documentation/userguide/docs/worksheets.md

```diff
@@ -55,4 +55,54 @@ delete buttons).
     for more information on AWS Athena SQL syntax.
 
 
+## :material-new-box: **Experimental Features: GenAI Powered Worksheets**
+
+As part of data.all >= v2.7 we introduced support for generative AI powered worksheet features. These features include both:
+
+1. Natural Language Querying (NLQ) of Structured Data
+2. Text Document Analysis of Unstructured Data
```

Comment:
maybe good to mention that it is only available for txt as well as pdf files

Reply:
have a warning in the `Text Document Analysis of Unstructured Data` Section for this